server/sonar-web/src/main/js/apps/organizations/navigation/__tests__/OrganizationNavigation-test.tsx

```diff
@@ -17,28 +17,35 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import React from 'react';
+import * as React from 'react';
 import { shallow } from 'enzyme';
 import OrganizationNavigation from '../OrganizationNavigation';
 
 jest.mock('../../../issues/utils', () => ({
   isMySet: () => false
 }));
 
+const baseOrganization = {
+  key: 'foo',
+  name: 'Foo',
+  canAdmin: false,
```

Comment:
Should be `isAdmin`

Reply:
Actually all these test don't make any sense here... I need to move them to inner components.